tajo-common/src/main/java/org/apache/tajo/exception/TajoException.java

```diff
@@ -27,6 +28,16 @@
 public class TajoException extends Exception implements TajoExceptionInterface {
   private ResultCode code;
 
+  public TajoException(ReturnState e) {
```

Comment:
```ReturnState``` contains stack trace as well as return code and message, but the stack trace is missed here. 
Would you give some backgrounds for this?